ensure/ensure.go

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package ensure provides a set of functions that panic if the error is not nil.
+package ensure
```

Comment:
I think we should almost never use this package.

Reply:
I'm not saying that we should replace `if err != nil`. This package is convenient for places where you know the things should not panic at all. Just a quick glance shows that we have at least 14 definitions of `must` and `try` in Talos and 15 those in Omni.

Reply:
Yes, I know, just they don't feel right to me neither - ideally they should also never panic, somehow handle error case. But I don't have a better suggestion, just an opinion.

Reply:
Panic is perfectly fine when you try to detect developer mistake. We do not have a lot of compile time checks and there is no way to execute code in compile time. So, in `init` blocks, for example, there is no better solution.